controlplane/kubeadm/internal/cluster.go

```diff
@@ -469,10 +469,6 @@ func (c *cluster) etcdIsHealthy(ctx context.Context) (healthCheckResult, error)
 		}
 	}
 
-	if len(response) > 0 {
-		return response, errors.New("could not check etcd member health")
-	}
-
```

Comment:
What happens if there are errors found, but the number of controlplane machines is equal to the known etcd members? It looks like we'll ignore errors now.

Reply:
the errors get embedded in the health check response type, we'll still have them in `response`.

that type is a map of nodes => errors encountered.

one sample might be:

```
{
  'my-first-control-plane': nil,
  'my-second-control-plane': errors.Errorf("etcd member has cluster ID %d, but all previously seen etcd members have cluster ID %d", clusterID, knownClusterID),
  'my-third-control-plane': errors.Errorf("etcd member reports alarms: %v", member.Alarms),
}
```

the `nil` in the return value of `response, nil` is an error not related to anything besides etcd errors (such as an api error when looking up control plane nodes)